src/common/session/session-entity.js

```diff
@@ -176,6 +179,7 @@ export class SessionEntity {
       if (this.isExpired(obj.expiresAt)) {
         this.collectSM('expired')
         this.collectSM('duration', obj, true)
+        this.state.numOfExpirations++
```

Comment:
This needs to get applied to the `inactive timer` as well since that can also cause a session to reset

Reply:
maybe just move the incrementer to the `reset()` call itself?